src/cli/restore/restore.go

```diff
@@ -34,15 +34,41 @@ func AddCommands(cmd *cobra.Command) {
 
 const restoreCommand = "restore"
 
+//nolint:lll
+const restoreCommandExamples = `# Restore emails to their original location
+corso restore exchange \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--destination '/'
+
+# Restore OneDrive to a folder named "my_latest_restore"
+corso restore onedrive \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--mail-folder '*' \
+	--destination my_latest_restore
+
+# Restore calendar events, restoring a copy if the event already exists.
+corso restore exchange \
+	--backup 1234abcd-12ab-cd34-56de-1234abcd \
+	--event-calendar '*' \
```

Comment:
Can we make this be a specific calendar example?

Reply:
Certainly